src/libraries/LPFeeLibrary.sol

```diff
@@ -9,8 +9,9 @@ library LPFeeLibrary {
     /// @notice Thrown when the static or dynamic fee on a pool exceeds 100%.
     error FeeTooLarge();
 
-    uint24 public constant STATIC_FEE_MASK = 0x7FFFFF;
+    uint24 public constant FEE_MASK = 0x7FFFFF;
     uint24 public constant DYNAMIC_FEE_FLAG = 0x800000;
+    uint24 public constant BEFORE_SWAP_FEE_OVERRIDE_FLAG = 0x800000;
```

Comment:
If we are going to use the same flag - we can just name this `FEE_FLAG` and have it defined once

Reply:
yeeah i did think that - just wasnt sure if the double naming makes the code more readable in other places

Reply:
hmm, i think if we want to use one, `DYNAMIC_FEE_FLAG` works

used for the poolkey and for fee-overrides, which makes sense to me

Reply:
consolidating to one constant, we might have 2 redundant functions:

```solidity
    function isDynamicFee(uint24 self) internal pure returns (bool) {
        return self & DYNAMIC_FEE_FLAG != 0;
    }
    
    /// @notice returns true if the fee has the override flag set (top bit of the uint24)
    function isOverride(uint24 self) internal pure returns (bool) {
        return self & DYNAMIC_FEE_FLAG != 0;
    }
 ```
 
 kind of like having `isOverride`, feels a lot clearer for
 
 ```solidity
         // if the beforeSwap hook returned a valid fee override, use that as the LP fee, otherwise load from storage
        uint24 lpFee =
            params.lpFeeOverride.isOverride() ? params.lpFeeOverride.removeOverrideAndValidate() : slot0Start.lpFee;
 ```